internal/docs/release.md

```diff
@@ -33,17 +65,18 @@ To do a release:
         past tense>`. For example, `**blob**: Added feature foo.`.
 -   Go to [Releases](https://github.com/google/go-cloud/releases). Click `Draft
     a new release`, enter your release name, select your tag from the dropdown,
-    and enter your release notes. Note that Github only has the last few commits
-    in the dropdown; if your commit is older than what it shows, you'll need to
-    create a tag from the command line (see below) and then select it in the UI.
-
-    ```bash
-    $ TAG=v0.n.0
-    $ COMMIT=aaaaaa
-    $ git tag -a ${TAG} -m ${TAG} ${COMMIT}
-    $ git push upstream ${TAG}
-    ```
+    and enter your release notes.
 
 -   Send an email to
     [go-cloud@googlegroups.com](https://groups.google.com/forum/#!forum/go-cloud)
     announcing the release, and including the release notes.
+
+-   Disable force pushing on GitHub.
+
+-   Add back `replace` lines:
+
+    ```bash
+    $ go run internal/releasehelper/releasehelper.go addreplace
+    ```
+
+    Run tests and send out PR as usual.
```

Comment:
Maybe add something about updating dependencies after the release?

Reply:
Done